pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -80,6 +80,11 @@ read_10x_files <- function(config, input_dir) {
       counts <- counts[[slot]]
     }
 
+    # Check if there are any rows with empty gene symbol in the count matrix and remove them if < 0.1% of the total number of features
+    if (length(which(rownames(counts) == "")) != 0 & length(which(rownames(counts) == ""))/nrow(counts) < 0.001) {
+      counts <- counts[!(rownames(counts) == ""),]
```

Comment:
given that you use this three times, I would extract the `which(rownames(counts) == "")` as a variable. this way, you perform the string comparation only once. 

Also, is the first condition really necessary? if the second one is true, the first one cannot be false.